python/telemetry/cute_70cm.py

```diff
@@ -32,25 +33,6 @@
     )
 
 
-class TimeAdapter(Adapter):
```

Comment:
Well spotted code duplication. It's probably my fault during PR reviews for PRs from this satellite team.